nautobot-app/{{ cookiecutter.project_slug }}/.github/workflows/ci.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.11"]
-        nautobot-version: ["{{cookiecutter.min_nautobot_version}}"]
+        nautobot-version: ["1.6.0"]
```

Comment:
The app may have a minimum version that's > `1.6.0` so I think this should remain the same. The default `min_nautobot_version` already appears to be `1.6.0`.

Reply:
Let's discuss this tomorrow.